pypeit/slittrace.py

```diff
@@ -1476,7 +1477,7 @@ def average_maskdef_offset(calib_slits, platescale, list_detectors):
     indx_b = np.where(np.in1d(calib_dets, spectrograph_dets[0]))[0]
     # if this spectrograph is not split into blue and red detectors
     # or if it is but there are no available offsets in the blue
-    if not blue_and_red or slitmask_offsets[indx_b].size == 0:
+    if not blue_and_red or indx_b.size == 0:
```

Comment:
Am wondering how this will go (or go away) once 
we start mosaicing..

Reply:
Good point! We may need to make some changes to this part of the code. I'll keep in mind.